target/linux/ath79/image/generic.mk

```diff
@@ -1492,6 +1492,24 @@ define Device/netgear_ex7300
 endef
 TARGET_DEVICES += netgear_ex7300
 
+define Device/netgear_ex7300v2
+  $(Device/netgear_generic)
+  SOC := qcn5502
+  DEVICE_MODEL := EX7300v2
+  UIMAGE_MAGIC := 0x27051956
+  NETGEAR_BOARD_ID := EX7300v2series
+  NETGEAR_HW_ID := 29765907+16+0+128
+  IMAGE_SIZE := 14528k
+  IMAGE/default := append-kernel | pad-offset $$$$(BLOCKSIZE) 64 | \
+	netgear-rootfs | pad-rootfs
+  IMAGE/sysupgrade.bin := $$(IMAGE/default) | append-metadata | \
+	check-size
+  IMAGE/factory.img := $$(IMAGE/default) | netgear-dni | \
+	check-size
```

Comment:
Same here, the header can probably be excluded for check-size ...

Reply:
Done